input/fsh/terminology.fsh

```diff
@@ -0,0 +1,376 @@
+* BCP47#ha "Hausa"
+* BCP47#haw "Hawaiian"
+* BCP47#he "Hebrew"
+* BCP47#hz "Herero"
+* BCP47#hil "Hiligaynon"
+* BCP47#hi "Hindi"
+* BCP47#hmn "Hmong"
+* BCP47#ho "Hiri Motu"
+* BCP47#hr "Croatian"
+* BCP47#hsb "Upper Sorbian"
+* BCP47#hu "Hungarian"
+* BCP47#hup "Hupa"
+* BCP47#hy "Armenian"
+* BCP47#iba "Iban"
+* BCP47#ig "Igbo"
+* BCP47#iii "Sichuan Yi"
+* BCP47#iu "Inuktitut"
+* BCP47#ilo "Iloko"
+* BCP47#id "Indonesian"
+* BCP47#inh "Ingush"
+* BCP47#ik "Inupiaq"
+* BCP47#is "Icelandic"
+* BCP47#it "Italian"
+* BCP47#jv "Javanese"
+* BCP47#ja "Japanese"
+* BCP47#jpr "Judeo-Persian"
+* BCP47#jrb "Judeo-Arabic"
+* BCP47#kaa "Kara-Kalpak"
+* BCP47#kab "Kabyle"
+* BCP47#kac "Kachin"
+* BCP47#kl "Kalaallisut"
+* BCP47#kam "Kamba (Kenya)"
+* BCP47#kn "Kannada"
+* BCP47#ks "Kashmiri"
+* BCP47#ka "Georgian"
+* BCP47#kr "Kanuri"
+* BCP47#kk "Kazakh"
+* BCP47#kbd "Kabardian"
+* BCP47#kha "Khasi"
+* BCP47#km "Khmer"
+* BCP47#ki "Kikuyu"
+* BCP47#rw "Kinyarwanda"
+* BCP47#kir "Kirghiz"
+* BCP47#kmb "Kimbundu"
+* BCP47#kok "Konkani"
+* BCP47#kv "Komi"
+* BCP47#kg "Kongo"
+* BCP47#ko "Korean"
+* BCP47#kos "Kosraean"
+* BCP47#kpe "Kpelle"
+* BCP47#krc "Karachay-Balkar"
+* BCP47#krl "Karelian"
+* BCP47#kru "Kurukh"
+* BCP47#kua "Kuanyama"
+* BCP47#kum "Kumyk"
+* BCP47#ku "Kurdish"
+* BCP47#kut "Kutenai"
+* BCP47#lad "Ladino"
+* BCP47#lah "Lahnda"
+* BCP47#lam "Lamba"
+* BCP47#lo "Lao"
+* BCP47#lv "Latvian"
+* BCP47#lez "Lezghian"
+* BCP47#lim "Limburgan"
+* BCP47#ln "Lingala"
+* BCP47#lt "Lithuanian"
+* BCP47#lol "Mongo"
+* BCP47#loz "Lozi"
+* BCP47#lb "Luxembourgish"
+* BCP47#lua "Luba-Lulua"
+* BCP47#lu "Luba-Katanga"
+* BCP47#lg "Ganda"
+* BCP47#lun "Lunda"
+* BCP47#luo "Luo (Kenya and Tanzania)"
+* BCP47#lus "Lushai"
+* BCP47#mad "Madurese"
+* BCP47#mag "Magahi"
+* BCP47#mh "Marshallese"
+* BCP47#mai "Maithili"
+* BCP47#mak "Makasar"
+* BCP47#ml "Malayalam"
+* BCP47#man "Mandingo"
+* BCP47#mr "Marathi"
+* BCP47#mas "Masai"
+* BCP47#mdf "Moksha"
+* BCP47#mdr "Mandar"
+* BCP47#men "Mende (Sierra Leone)"
+* BCP47#mic "Mi'kmaq"
+* BCP47#min "Minangkabau"
+* BCP47#mk "Macedonian"
+* BCP47#mg "Malagasy"
+* BCP47#mt "Maltese"
+* BCP47#mnc "Manchu"
+* BCP47#mni "Manipuri"
+* BCP47#moh "Mohawk"
+* BCP47#mn "Mongolian"
+* BCP47#mos "Mossi"
+* BCP47#mri "Maori"
+* BCP47#ms "Malay"
+* BCP47#mus "Creek"
+* BCP47#mwl "Mirandese"
+* BCP47#mwr "Marwari"
+* BCP47#my "Burmese"
+* BCP47#myv "Erzya"
+* BCP47#nap "Neapolitan"
+* BCP47#na "Nauru"
+* BCP47#nv "Navajo"
+* BCP47#nbl "South Ndebele"
+* BCP47#nde "North Ndebele"
+* BCP47#ng "Ndonga"
+* BCP47#nds "Low German"
+* BCP47#ne "Nepali"
+* BCP47#new "Newari"
+* BCP47#nia "Nias"
+* BCP47#niu "Niuean"
+* BCP47#nl "Dutch"
+* BCP47#nn "Norwegian Nynorsk"
+* BCP47#nb "Norwegian Bokmål"
+* BCP47#nog "Nogai"
+* BCP47#no "Norwegian"
+* BCP47#nqo "N'Ko"
+* BCP47#nso "Pedi"
+* BCP47#nya "Nyanja"
+* BCP47#nym "Nyamwezi"
+* BCP47#nyn "Nyankole"
+* BCP47#nyo "Nyoro"
+* BCP47#nzi "Nzima"
+* BCP47#oci "Occitan (post 1500)"
```

Comment:
sorry this is my bad. I've been meaning to remove the ` (post 1500)` for a while. We don't include the ISO years.

Reply:
Done in https://github.com/SaraAlert/saraalert-fhir-ig/pull/11/commits/1ed8c290ef5c6fa4e58fae9747fee73f3504d270.